locale/English_en.ftl

```diff
@@ -538,8 +538,8 @@ game_selected_changed_on_opening = Game Selected changed to {"{"}{"}"}, as the P
 ### Extra stuff I don't remember where it goes.
 
 rpfm_title = Rusted PackFile Manager
-delete_mymod_0 = <p>You are about to delete this <i>'MyMod'</i> from your disk.</p><p>There is no way to recover it after that.</p><p>Are you sure?</p>
-delete_mymod_1 = <p>There are some changes yet to be saved.</p><p>Are you sure?</p>
+delete_mymod_0 = <p>Are you sure to delete this <i>'MyMod'</i> from your disk.</p><p>There is no way to recover it after that.</p>
```

Comment:
About this line, it already clearly says what's going to happen. I see no need for changing it.

Reply:
It's shorter 😄 

Seriously, "Are you sure?" by itself is unclear. You have to read the context to know what's going on. 

Take delete_mymod_0. Are you sure there is no way to recover, or are you sure you are about to delete this? Neither.

Reply:
Ok then. Just needs a couple of changes then:
- Put "Are you sure you want..." instead of "Are you sure...", as it sounds a bit better (same for the other instances of this).
- Change "from your disk." with "from your disk?". There's a missing question mark there.